src/main/java/com/spring/task/gymcrm/utils/TraineeMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.spring.task.gymcrm.utils;
+
+import com.spring.task.gymcrm.dto.TraineeDto;
+import com.spring.task.gymcrm.entity.Trainee;
+import com.spring.task.gymcrm.entity.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class TraineeMapper {
```

Comment:
Great class!

P.s. I think the best place for all mappers is 'entity.mapper' package. What do you think?